site/pages/signer/authentication/social-login.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Oauth Authentication
+description: Authenticate a user using Oauth
+---
+
+# Social Login Authentication
```

Comment:
can we please add an explanation (perhaps a separate doc?) about how to bring other auth provides such as Twitter/Discord via Auth0? I want to make sure developers who need those auth providers are able to find an obvious answer and guide in our docs!